### PR TITLE
User Page

### PR DESCRIPTION
**Pull request by LissaGreense**
I added User page, but statistics are still placeholders :)

### CODE REVIEW

File: frontend/src/pages/accountPage.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+export const AccountPage = () => {
+    const navigate = useNavigate();
+    const backend_url = 'http://localhost:8000';
+    const [avatar, setAvatar] = useState<string>('');
+    const [showCropper, setShowCropper] = useState<boolean>(false);
+    const [currentImage, setCurrentImage] = useState<string | ArrayBuffer | undefined>();
+    const [croppedAreaPixels, setCroppedAreaPixels] = useState<Area | null>(null);
+    const [crop, setCrop] = useState<{ x: number; y: number }>({ x: 0, y: 0 });
+    const [zoom, setZoom] = useState(1)
```
Comment (adlbrtbsr) [inline frontend/src/pages/accountPage.tsx:22]: can you add a type?

Comment (LissaGreense) [reply frontend/src/pages/accountPage.tsx]: done :D

File: frontend/src/pages/accountPage.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                            objectFit="vertical-cover"
+                            onCropChange={setCrop} crop={crop} zoom={zoom}
+                            onCropComplete={onCropComplete}
+                            onZoomChange={setZoom}
+                        />
+                    )
+                    }
+                </div>
+                <div className="crop-controls">
+                    <Button onClick={handleUploadImage}>Zaladuj</Button>
```
Comment (adlbrtbsr) [inline frontend/src/pages/accountPage.tsx:82]: minor spelling mistake 'Zaladuj'

Comment (LissaGreense) [reply frontend/src/pages/accountPage.tsx]: done :)

**Review by adlbrtbsr — APPROVED** [review]
good job :)

**Pull request merged**